src/new/sendTransaction.ts

```diff
@@ -92,6 +95,7 @@ export default class SendTransaction extends EventEmitter {
 
   /**
    *
+   * @param {HathorWallet} wallet Wallet instance
    * @param {IStorage} storage Storage object
```

Comment:
suggestion(non-blocking): Make it clear that this storage will be discarded if the wallet parameter is present
```suggestion
   * @param {IStorage} storage Storage object, superseded by `wallet.storage` if wallet is present
```